pkgs/applications/networking/instant-messengers/ytalk/default.nix

```
@@ -0,0 +1,21 @@
+{ stdenv, fetchurl, ncurses }:
+
+stdenv.mkDerivation rec {
+  name = "ytalk-${version}";
+  version = "3.3.0";
+
+  src = fetchurl {
+    url = "ftp://ftp.ourproject.org/pub/ytalk/${name}.tar.gz";
+    sha256 = "1d3jhnj8rgzxyxjwfa22vh45qwzjvxw1qh8fz6b7nfkj3zvk9jvf";
+  };
+
+  buildInputs = [ ncurses ];
+
+  meta = {
+    homepage    = http://ytalk.ourproject.org;
```

```suggestion
    homepage = "http://ytalk.ourproject.org";
```
URLs should be quoted as per https://github.com/NixOS/rfcs/pull/45